src/perception/vision_pipeline/vision_pipeline/node_detector.py

```diff
@@ -158,7 +93,11 @@ def cone_msg(
 
 
 class DetectorNode(Node):
-    def __init__(self):
+    def __init__(
+        self, 
+        mode: int, # mode of detection. 0==cv2, 1==torch, 2==trt
```

Comment:
use an enum for this

Reply:
or, based on usages below - probably doesn't even need to be multiple options. Change to be some sort of boolean flag `enable_cv_filters`?

Reply:
i have the mode just for IDing which mode. it can be a string to print with the node initialised message

Reply:
If u want to have each mode passed in there, make it an enum